examples/constraint-explorer.rs

```diff
@@ -123,7 +132,7 @@ impl App {
     }
 
     fn handle_events(&mut self) -> Result<()> {
-        use KeyCode::*;
+        use KeyCode::{Char, Down, Esc, Left, Right, Up};
```

Comment:
I don't mind this - it's good enough and I think it probably suffers the same edge case that can make people wonder what `Char` is. The others are a bit more obvious, but `Char` having the same name as `char` seems enough to disambiguate like this to me.

Reply:
The I'd prefer to inline this when this results in the same amount of lines → `match … { KeyCode::Esc =>  }`
Then its even more explicit what it is and also the way its normally written / auto generated by rust-analyzer.

Reply:
done

Reply:
I've left the constraint / layout / flex examples with imports for `Constraint::{...}` because the quantity of usages of Constraint there is overwhelming and obvious.